docs/source/commands/configtxgen.md

```diff
@@ -86,16 +86,19 @@ for channel reconfiguration workflows, such as adding a member).
 configtxgen -printOrg Org1
 ```
 
-### Output anchor peer tx
-
-Output a configuration update transaction to `anchor_peer_tx.pb` which sets the
-anchor peers for organization Org1 as defined in profile
-SampleSingleMSPChannelV1_1 based on `configtx.yaml`.
+### Output anchor peer tx (deprecated)
 
+Output a channel configuration update transaction `anchor_peer_tx.pb`  based on
+the anchor peers defined for Org1 and channel profile SampleSingleMSPChannelV1_1
+in `configtx.yaml`. Transaction will set anchor peers for Org1 if no anchor peers
+have been set on the channel.
 ```
 configtxgen -outputAnchorPeersUpdate anchor_peer_tx.pb -profile SampleSingleMSPChannelV1_1 -asOrg Org1
 ```
 
+This transaction has been deprecated. To set anchor peers on the channel, use
```

Comment:
The 'transaction' hasn't been deprecated, the command flag -outputAnchorPeersUpdate that generates the config transaction has been deprecated.

Reply:
Changed to `The `-outputAnchorPeersUpdate` output flag has been deprecated.`